cpp/test.h

```diff
@@ -89,15 +89,15 @@ class Watchdog : public lunchbox::Thread
             lunchbox::Thread::setName( "Watchdog" );
 #ifdef TEST_RUNTIME
             lunchbox::sleep( TEST_RUNTIME * 1000 );
-            TESTINFO( false,
-                      "Watchdog triggered - " << _name <<
-                      " did not terminate within " << TEST_RUNTIME << "s" );
+            std::cerr << "Watchdog triggered - " << _name
```

Comment:
Because it was just a hack to get the abort for free, which is now called explicitly.